src/TranslateExtension.php

```diff
@@ -6,6 +6,7 @@
 use Bolt\Asset\Target;
 use Bolt\Controller\Zone;
 use Bolt\Extension\Animal\Translate\Config;
+use Bolt\Extension\Bacboslab\Menueditor\Event\FieldBuilderEvent;
```

Comment:
This class (FieldBuilderEvent) does not seem to be in this PR or in the current repo, could you add it?